### DISCUSSION TITLE
Why the lates version has "experimental" downloader

### DISCUSSION THREAD

**Discussion by harrybutts** [aclap-dev/video-downloadhelper#45] [Help and Q&A]
### Your question

Why does the lates version have an "experimental" downloader that "side downloads"? If I wanted to be experimental I would be on the beta version.  Any way to get off of this experimental crap?

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Blank Blank",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
Hello Harry.

> Any way to get off of this experimental crap?

We're not engaging with rude comments.

I'm blocking this thread.
